src/server/types.ts

```diff
@@ -19,6 +19,14 @@ export interface FreshOptions {
   render?: RenderFunction;
   plugins?: Plugin[];
   staticDir?: string;
+  router?: RouterOptions;
+}
+
+export interface RouterOptions {
+  /**
+   *  @default [false]
```

Comment:
```suggestion
   *  @default {false}
```
The `@default` tag takes curly braces. Example [here](https://deno.land/std@0.191.0/async/mod.ts?s=DelayOptions).

Could you please add a small description for this property too?